File: data/sample/atcoder/abc105_b/C/impl_03.c
#include <stdio.h>

int main(){
  int N;
  scanf("%d",&N);
  for (int i = 0; i * 4 <= N; i++){
    int z = N - (i * 4);
    if (z % 7 == 0){
      printf("Yes");
      return 0;
    }
  }
  printf("No");
  return 0;
}

File: data/sample/atcoder/abc105_b/Rust/impl_03.rs
use proconio::input;

fn main() {
    input! {
        n: usize
    }

    for i in 0.. {
        if i * 4 > n {
            break;
        }
        if (n - i * 4) % 7 == 0 {
            println!("Yes");
            return;
        }
    }
    println!("No");
}
